Dockerfile

```diff
@@ -17,7 +17,7 @@ ARG AZURITE_VERSION=3.18.0
 ARG GOTESTSUM_VERSION=v1.9.0
 ARG DELVE_VERSION=v1.22.1
```

Comment:
Do we need to update this one? I recall we had to update it in moby at some point due to incompatibility with go versions; https://github.com/moby/moby/pull/48497

current version looks to be 1.23.1; https://github.com/go-delve/delve/releases/tag/v1.23.1

Reply:
I also went looking for other related things, and was confused for a bit for the `gopls` version, because there's to v0.26, but it looks like we're using the `golang.org/x/tools` tag instead of the `gopls/vXXX` tags; https://github.com/moby/buildkit/blob/ca01cabdfca79b54deccdf779a2c3bc60da888ae/hack/dockerfiles/lint.Dockerfile#L8

Wondering if we should change that (https://github.com/golang/tools/releases/tag/gopls%2Fv0.16.2)

Reply:
> Do we need to update this one?

Ah yes we should, there are fixes for Go 1.23 in latest stable, let me do that.

> but it looks like we're using the `golang.org/x/tools` tag instead of the `gopls/vXXX` tags

I don't think we want specifically the gopls tool https://github.com/golang/tools/tree/master/gopls but tool repo to handle extra analyzers. Maybe `GOTOOL_VERSION` would be a better name.

Reply:
> Maybe GOTOOLS_VERSION would be a better name

That works as well 😅 I was just a bit confused "what's this version?" 😂